server/modules/user/api/userApi.js

```diff
@@ -26,7 +26,8 @@ module.exports.init = app => {
         throw new SystemError('invalidUser')
       }
 
-      await UserService.inviteUser(user, surveyId, email, groupUuid)
+      const serverUrl = `${req.protocol}://${req.headers.host}`
```

Comment:
Should we add a method to Request? e.g. ```Request.getServerUrl(req)``` ?
I would also add the port

Reply:
Ok added. Chrome, Firefox and Safari seem to add the port number to the request header